google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/InstanceAdmin.java

```diff
@@ -0,0 +1,213 @@
+    } catch (Exception e) {
+      System.out.println("Error listing instances: " + e.getMessage());
+    }
+    // [END bigtable_list_instances]
+
+    System.out.println(); //for a new-line
+    System.out.println("Get Instance:");
+    // [START bigtable_get_instance]
+    try {
+      Instance instance = adminClient.getInstance(instanceID);
+      System.out.println("Instance ID: " + instance.getId());
+      System.out.println("Instance Meta:");
+      System.out.println("Display Name: " + instance.getDisplayName());
+      System.out.println("Labels:");
+      Map<String, String> labels = instance.getLabels();
+      for (String key : labels.keySet()) {
+        System.out.println(key + ": " + labels.get(key));
+      }
+      System.out.println("State: " + instance.getState());
+      System.out.println("Type: " + instance.getType());
+    } catch (Exception e) {
+      System.out.println("Error getting instance: " + e.getMessage());
+    }
+    // [END bigtable_get_instance]
+
+    System.out.println(); //for a new-line
+    System.out.println("Listing Clusters...");
+    // [START bigtable_get_clusters]
+    try {
+      List<Cluster> clusters = adminClient.listClusters(instanceID);
+      for (Cluster cluster : clusters) {
+        System.out.println(cluster.getId());
+      }
+    } catch (Exception e) {
+      System.out.println("Error listing clusters: " + e.getMessage());
+    }
+    // [END bigtable_get_clusters]
+  }
+
+  public static void createDevInstance(BigtableInstanceAdminClient adminClient, String instanceID,
+      String clusterID) {
+    // [START bigtable_create_dev_instance]
+    System.out.println(); //for a new-line
+    System.out.println("Creating a DEVELOPMENT Instance");
+    // Creates a Development instance with the ID "hdd-instance"
+    // with cluster ID "hdd-cluster" and location us-central1-f
+    // Cluster nodes should not be set while creating Development Instance
+    CreateInstanceRequest createInstanceRequest = CreateInstanceRequest.of(instanceID)
+        .addCluster(clusterID, "us-central1-f", 1, StorageType.HDD).setType(Type.DEVELOPMENT)
+        .addLabel("dev-label", "dev-label");
+    // Create development instance with given request
+    try {
+      Instance instance = adminClient.createInstance(createInstanceRequest);
+      System.out
+          .println("DEVELOPMENT type instance : " + instance.getId() + " created successfully");
+    } catch (Exception e) {
+      System.out.println("Error creating dev-instance: " + e.getMessage());
+    }
+    // [END bigtable_create_dev_instance]
+  }
+
+  // Delete the Instance
+  public static void deleteInstance(BigtableInstanceAdminClient adminClient, String instanceID) {
+    // [START bigtable_delete_instance]
+    System.out.println(); //for a new-line
+    System.out.println("Deleting Instance");
+    try {
+      adminClient.deleteInstance(instanceID);
+      System.out.println("Instance deleted: " + instanceID);
+    } catch (Exception e) {
```

Comment:
Would be good to explicit catch a typesafe exception to demonstrate their usage. Maybe catch the NotFound exception?